js/packages/openinference-instrumentation-langchain/src/utils.ts

```diff
@@ -18,9 +20,14 @@ import {
   LLMMessageFunctionCall,
   LLMMessageToolCalls,
   LLMMessagesAttributes,
+  LLMOpenInferenceAttributes,
+  RetrievalDocument,
   SafeFunction,
 } from "./types";
 
+export const RETRIEVAL_DOCUMENTS =
+  `${SemanticAttributePrefixes.retrieval}.${RetrievalAttributePostfixes.documents}` as const;
```

Comment:
not really a util, I'd put it closer to where you are using it.

Reply:
I think this should just be added to the semantic conventions package, will open a pr.